src/__test__/__mocks__/mocksAPI.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-console */
+const axios = require('axios');
+
+function getCoins() {
+  axios.get('https://api.coincap.io/v2/assets?limit=1')
+    .then((response) => console.log(response.data))
+    .catch((error) => console.log(error));
+}
```

Comment:
- [ ] Change the syntax for this function and use instead the ES6 syntax i.e arrow function.
- [ ] Add a test case for the `getOneCoin` function as well.

Reply:
Hey there, in here I'm testing that part.
![image](https://user-images.githubusercontent.com/74747182/137321423-1173a1b2-cfc4-417e-8e14-35c3443f0ebd.png)
So I'm testing it with a console.log(), when I run this test it takes that function `getCoins()` and return me the data.